cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -266,28 +267,24 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
 
             LOG.info(f"RNA coverage bigwig file {rna_coverage_bigwig.path} found.")
```

Comment:
```suggestion
            LOG.debug(f"RNA coverage bigwig file {rna_coverage_bigwig.path} found.")
```
Maybe we can move this to debug to keep the output cleaner? I think @Vince-janv  was trying to clean up some of this. The log below indicates that files have been found also.